composer.json

```diff
@@ -29,6 +29,7 @@
         "symfony/routing": "^2.3 || ^3.0"
     },
     "suggest": {
+        "doctrine/orm": "^2.0",
```

Comment:
In the test I use `Doctrine\ORM\Query` and `Doctrine\ORM\EntityManagerInterface`, how do if I remove this line ?